src/content/configuration/module.md

```diff
@@ -308,12 +308,26 @@ module.exports = {
 
 ## `Rule.use`
 
-A list of [UseEntries](#useentry) which are applied to modules. Each entry specifies a loader to be used.
+`[UseEntry] | function`
+
+`Rule.use` can be an array of [UseEntry](#useentry) which are applied to modules. Each entry specifies a loader to be used.
 
 Passing a string (i.e. `use: [ 'style-loader' ]`) is a shortcut to the loader property (i.e. `use: [ { loader: 'style-loader '} ]`).
 
 Loaders can be chained by passing multiple loaders, which will be applied from right to left (last to first configured).
 
+`Rule.use` can be a function which receives the object argument describing the module being loaded, and must return an array of `UseEntry`.
+
+The object parameter has the following fields:
+
+- `compiler`: The current webpack compiler (can be undefined)
+- `issuer`: The path to the module that is importing the module being loaded
+- `resource`: The path to the module being loaded
+- `realResource`: The path to the module being loaded
```

Comment:
It is worth to note what differs `resource` and `realResource` here as it may confuse the reader @bew

Reply:
Indeed that would be better, but as stated in the comments above, I don't actually know what is the difference ^^
If you can help about this I'd be very grateful :)

Reply:
`resource` is usually equal `realResource` expect when the `resource` name is overwritten via `!=!` in request string. `realResource` is always the resource file which is requested.

`require("LLL!RRR")` -> `resource: RRR, realResource: RRR`
`require("MMM!=!LLL!RRR")` -> `resource: MMM, realResource: RRR`

Reply:
Thanks, I'll add a note when I'm at work